spec/models/room_spec.rb

```diff
@@ -54,12 +54,41 @@
     end
   end
 
-  describe 'after_create' do
-    describe 'create_meeting_options' do
-      it 'creates a RoomMeetingOption for each MeetingOption' do
-        create_list(:meeting_option, 5)
+  describe 'create_meeting_options' do
+    let!(:meeting_option) { create(:meeting_option, name: 'glViewerAccessCode', default_value: '') }
 
-        expect { create(:room) }.to change(RoomMeetingOption, :count).from(0).to(5)
+    before do
+      create_list(:meeting_option, 5)
+    end
+
+    it 'creates a RoomMeetingOption for each MeetingOption' do
+      expect { create(:room) }.to change(RoomMeetingOption, :count).from(0).to(6)
+    end
+
+    it 'does not generate an access code if the room config is not enabled' do
```

Comment:
Access code feels like a test group, we can group those examples under a `context` maybe?

Reply:
I put the after_create context, and upon review I dont think an extra context is necessary